db/c_test.c

```diff
@@ -1276,6 +1276,9 @@ int main(int argc, char** argv) {
     CheckPinGetCF(db, roptions, handles[1], "box", "c");
     rocksdb_writebatch_destroy(wb);
 
+    rocksdb_flush_wal(db, 1, &err);
+    CheckNoError(err)
```

Comment:
Nit: Please put a ";" at the end of line

Reply:
Do you mean the `manual_wal_flush`? If so, I think that test only tests the setters and getters; the options object is destroyed afterwards.